src/Native/libunwind/include/__libunwind_config.h

```diff
@@ -44,8 +44,8 @@
 #    define _LIBUNWIND_CONTEXT_SIZE 76
 #    define _LIBUNWIND_CURSOR_SIZE 83
 #  else
-#    define _LIBUNWIND_CONTEXT_SIZE 58
-#    define _LIBUNWIND_CURSOR_SIZE 65
+#    define _LIBUNWIND_CONTEXT_SIZE 50
```

Comment:
I would not expect changes in libunwind. @janvorli What do you think about this change?